docs/ref-bundle-stages.md

```diff
@@ -17,3 +17,40 @@ To demonstrate the life cycle of a Fleet bundle, we will use [multi-cluster/helm
 This diagram shows the different rendering stages a bundle goes through until deployment.
 
 ![Bundle Stages](/img/FleetBundleStages.svg)
+
+## Examining the Bundle Lifecycle With the CLI
+
+Several fleet CLI commands help with debugging bundles.
+
+### fleet apply
+
+[Apply](./cli/fleet-cli/fleet_apply.md) renders a folder with Kubernetes resources, such as a Helm chart, manifests, or kustomize folders, into a Fleet bundle resource.
+
```

Comment:
nit: I feel that a kind of introductory comment to this code block would make sense. WDYT?
```suggestion

Here is an example of how to use it:
```